client/dashboard/task-list/tasks/appearance.js

```diff
@@ -48,6 +48,13 @@ class Appearance extends Component {
 		this.updateNotice = this.updateNotice.bind( this );
 	}
 
+	componentDidMount() {
+		const { from } = getQuery();
+		if ( 'homepage' === from ) {
```

Comment:
I think we should also remove this query from the URL after recording the event, just to avoid accidentally re-triggering this event.

Reply:
I ended up implementing a tracks queue in a separate PR that removes this:
https://github.com/woocommerce/woocommerce-admin/pull/3250